src/elements/content-sidebar/TaskModal.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import BetaFeedbackBadge from '../../features/beta-feedback';
+import Modal from '../../components/modal/Modal';
+import TaskForm from './activity-feed/task-form';
+import messages from '../common/messages';
+import { TASK_TYPE_APPROVAL, TASK_TYPE_GENERAL } from '../../constants';
+
+type TaskModalProps = {
+    error: ?ElementsXhrError,
+    feedbackUrl: string,
+    handleCreateError: Function,
+    handleCreateSuccess: Function,
+    handleModalClose: Function,
+    isTaskFormOpen: boolean,
+    passThrough: Object,
+    taskType: TaskType,
+};
+
+function getMessageForModalTitle(taskType: TaskType): Object {
+    switch (taskType) {
+        case TASK_TYPE_GENERAL:
```

Comment:
Minor, but it seems like this could use an object literal instead of switch/case.

Reply:
there will be a little more logic here soon, so will update when that lands